client/deployments/client.go

```diff
@@ -296,6 +297,24 @@ func (c *Client) DirectUpload(
 		return errors.Wrap(err, "Cannot read artifact file stats")
 	}
 
+	tr := tar.NewReader(artifact)
+	versionH, err := tr.Next()
+	if err != nil {
+		return errors.Wrap(err, "Cannot find artifact")
```

Comment:
Can we adjust the error message a little?
I found the current message quite confusing when I was testing the CLI.
```suggestion
		return errors.Wrap(err, "error parsing artifact")
```